### PR TITLE
Feature/DEV-7347.fix bug in date or cut label

### PR DESCRIPTION
**Pull request by thomasdotred**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [ ] Tests pass

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: lusidtools/cocoon/cocoon.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def load_quote_batch(
             )
 
         return api_factory.build(lusid.api.QuotesApi).upsert_quotes(
-            scope=kwargs["scope"],
+            scope=kwargs["scope"],
```
Comment (slemasne) [inline lusidtools/cocoon/cocoon.py]: What is changing here, looks like you're adding a whitespace?

File: lusidtools/cocoon/dateorcutlabel.py [hunk 2]
```diff
@@ -7,72 +7,202 @@
 from collections import UserString
 
 
+def _process_timestamp(datetime_value: pd.Timestamp):
+    """
+    Processes pandas timestamp convert it to ISO format and parse to string
+
+    Parameters
+    ----------
+    datetime    pd.Timestamp
```
Comment (slemasne) [inline lusidtools/cocoon/dateorcutlabel.py]: Should parameter docstring follow format used elsewhere in the project?

_:param pd.Timestamp datetime_value:_

Comment (thomasdotred) [reply lusidtools/cocoon/dateorcutlabel.py]: see numpy post on #client-engineering

File: lusidtools/cocoon/cocoon.py (hunk 1 above)
Comment (thomasdotred) [reply lusidtools/cocoon/cocoon.py]: Ensuring black formatting happens

**Pull request merged**